garden_ai/gardens.py

```diff
@@ -62,6 +87,10 @@ def __getattr__(self, name):
             elif name == short_name:
                 message_extra = f" Did you mean {alias}?"
 
+        for modal_function in self.modal_functions:
+            if name == modal_function.metadata.function_name:
+                return modal_function
```

Comment:
Something I'm wondering is how we'll handle methods on Modal classes. I think the backend should handle this by just storing the dotted function name (eg, `my_class.my_method`) but that will probably be weird for the SDK.

Reply:
talked about this on a call but for posterity: I think that we'll have a `ModalClass` analogue to `ModalFunction` (so Gardens will have entrypoints, modal functions, and modal classes) and a `ModalClass.__getattr__` implementation *should* get us the intuitive syntax of `my_garden.my_modal_class.my_modal_method` without much extra legwork